src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1019.RuleIdMustBePresentAndConsistent_Invalid.sarif

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Sarif Functional Testing"
+        }
+      },
+      "results": [
+        {
```

Comment:
>{ [](start = 8, length = 1)

Our convention is that the _Valid file contains only valid results, and the _Invalid file contains only invalid results. This result is valid, so remove it from the _Invalid file.